src/gui/PianoView.cpp

```diff
@@ -36,98 +36,94 @@
  * \todo write isWhite inline function and replace throughout
  */
 
-#include <cmath>
+#include "PianoView.h"
 
 #include <QCursor>
 #include <QKeyEvent>
 #include <QPainter>
 #include <QVBoxLayout>
+#include <cmath>
 
-#include "PianoView.h"
-#include "Piano.h"
 #include "CaptionMenu.h"
-#include "embed.h"
-#include "gui_templates.h"
 #include "InstrumentTrack.h"
 #include "Knob.h"
-#include "StringPairDrag.h"
 #include "MainWindow.h"
-
+#include "Piano.h"
+#include "StringPairDrag.h"
+#include "embed.h"
+#include "gui_templates.h"
 
 /*! The scale of C Major - white keys only.
  */
 Keys WhiteKeys[] =
-{
-	Key_C, Key_D, Key_E, Key_F, Key_G, Key_A, Key_H
-} ;
-
-
-QPixmap * PianoView::s_whiteKeyPm = NULL;           /*!< A white key released */
-QPixmap * PianoView::s_blackKeyPm = NULL;           /*!< A black key released */
-QPixmap * PianoView::s_whiteKeyPressedPm = NULL;    /*!< A white key pressed */
-QPixmap * PianoView::s_blackKeyPressedPm = NULL;    /*!< A black key pressed */
+	{
+		Key_C, Key_D, Key_E, Key_F, Key_G, Key_A, Key_H};
 
+QPixmap* PianoView::s_whiteKeyPm = NULL;		/*!< A white key released */
+QPixmap* PianoView::s_blackKeyPm = NULL;		/*!< A black key released */
+QPixmap* PianoView::s_whiteKeyPressedPm = NULL; /*!< A white key pressed */
+QPixmap* PianoView::s_blackKeyPressedPm = NULL; /*!< A black key pressed */
 
-const int PIANO_BASE = 11;          /*!< The height of the root note display */
-const int PW_WHITE_KEY_WIDTH = 10;  /*!< The width of a white key */
-const int PW_BLACK_KEY_WIDTH = 8;   /*!< The width of a black key */
+const int PIANO_BASE = 11;			/*!< The height of the root note display */
```

Comment:
What's happening here?

Reply:
It's trying to indent the trailing comments, but it wants to do that with tabs (probably because our normal indentation is also tabs). I think it's actually logical - If we indent all (including non-trailing comments) with tabs - why not trailing comments?

Alternatives are trying to use spaces (trying `SpacesBeforeTrailingComments`, not sure if this will work) disabling trailing comments (setting `AlignTrailingComments` to false).

Reply:
I'm fine with that indentation. Resolving. Feel free to comment against and reopen the discussion.